### PR TITLE
missing checkpoint to instance.start

### PR DESCRIPTION
**Pull request by alena1108**
@cjellick

### CODE REVIEW

File: code/iaas/logic/src/main/java/io/cattle/platform/process/instance/InstanceStart.java [hunk 1]
```diff
@@ -151,7 +152,7 @@ public HandlerResult handle(ProcessState state, ProcessInstance process) {
         return result;
     }
 
-    protected void waitForDeploymentUnit(Instance instance) {
+    protected void waitForDeploymentUnitAllocate(Instance instance) {
         // Wait until all instances in the deployment unit are out of the creating state (to ensure all of instnace.create has ran)
         if(StringUtils.isEmpty(instance.getDeploymentUnitUuid())) {
             return;
```
Comment (cjellick) [inline code/iaas/logic/src/main/java/io/cattle/platform/process/instance/InstanceStart.java]: Would it make sense to move the checkpoint to below this short circuit?

Comment (alena1108) [inline code/iaas/logic/src/main/java/io/cattle/platform/process/instance/InstanceStart.java]: I just wanted to keep all the checkpoints in the same method (handle()) for better visibility

File: code/iaas/logic/src/main/java/io/cattle/platform/process/instance/InstanceStart.java [hunk 2]
```diff
@@ -105,8 +105,9 @@ public HandlerResult handle(ProcessState state, ProcessInstance process) {
                 // wait until volumesFrom/networksFrom containers start up
                 waitForDependenciesStart(instance);
 
+                progress.checkPoint("Waiting for deployment unit instances to allocate");
```
Comment (cjellick) [inline code/iaas/logic/src/main/java/io/cattle/platform/process/instance/InstanceStart.java]: We aren't waiting for them to be allocated, just to be past creating. If anything, the constant UNALLOCATED_WAIT_STATES is misnamed.

This comment applies to the method name changes as well.

Comment (alena1108) [inline code/iaas/logic/src/main/java/io/cattle/platform/process/instance/InstanceStart.java]: @cjellick how about waitForDeploymentUnitCreate as a method name? just "wait" is a bit unclear

Comment (cjellick) [inline code/iaas/logic/src/main/java/io/cattle/platform/process/instance/InstanceStart.java]: Yes @alena1108, I like that.

**Comment by cjellick** [comment]
LGTM

**Pull request merged**